### ISSUE TITLE
Could not extract version from api.js

### ISSUE THREAD

**Issue by F3nd0** [taylordotfish/librecaptcha#9]
I've been trying to use librecaptcha to complete a registration on [this page](http://forum.world.st/template/NamlServlet.jtp?start_registration_page), using the following command:

```
librecaptcha --gui '6Lf12z4UAAAAAKFG8EczAb6BjVvaFM2rqHswrCP7' 'http://forum.world.st' 'Mozilla/5.0 (X11; Linux x86_64; rv:68.0) Gecko/20100101 Firefox/68.0'
```

However, the above command crashes with the following error:

```
An unexpected error occurred. The exception traceback is shown below:

Traceback (most recent call last):
  File "/usr/bin/librecaptcha", line 10, in <module>
    sys.exit(main())
  File "/usr/lib/python3.8/site-packages/librecaptcha/__main__.py", line 261, in main
    run_or_exit(parsed)
  File "/usr/lib/python3.8/site-packages/librecaptcha/__main__.py", line 229, in run_or_exit
    return run(args)
  File "/usr/lib/python3.8/site-packages/librecaptcha/__main__.py", line 207, in run
    uvtoken = get_token(
  File "/usr/lib/python3.8/site-packages/librecaptcha/librecaptcha.py", line 46, in get_token
    rc = ReCaptcha(api_key, site_url, user_agent, debug=debug)
  File "/usr/lib/python3.8/site-packages/librecaptcha/recaptcha.py", line 319, in __init__
    self.rc_version = get_rc_version(self.user_agent)
  File "/usr/lib/python3.8/site-packages/librecaptcha/recaptcha.py", line 136, in get_rc_version
    raise RuntimeError("Could not extract version from api.js.")
RuntimeError: Could not extract version from api.js.
```

**Comment by HK-Mattew** [comment]
I also have this error.

**Comment by taylordotfish** [comment]
Fixed in version 0.6.3.
